### ISSUE TITLE
1.1 release

### ISSUE THREAD

**Issue by Bertram25** [ValyriaTear/ValyriaTear#554]
This issue is there for general discussion purpose about the 1.1 release, which should be the next one.

**Comment by Bertram25** [comment]
@gunchleoc @RyDroid : Hi again guys, I just opened this issue so we can share each others thought about the current dev effort right here!

I wanted to add a few notes there so that everybody is aware of what is happening, even if I'm super slow with progress atm, but it's improving. ;)

**Just a side note about the project feature and code:**
As I've seen @gunchleoc open bugs in Hero of Allacrost, I wanted to add this:
Valyria Tear is originally a fork from HoA. But, please note that Valyria and Hero of Allacrost code are more and more far away from each other, so for the sole purpose of telling it once, don't expect features in Hoa to be present in VT and the reverse.
We also don't have the same scope and it's good, since this will make 2 worth playing games instead of 2 similar stuff around.

**Current effort:**
As you or may not seen, I've starting improving the gameplay in the 'new_game_balance' branch, which is now also containing bugfixes and graphics updates. My goal is to add a few more features and changes, like escape circles so that the players can go back to some safe place and buy items in the beginning of the game, make items buyable through money, but also trades, and check the characters can improve their equipment more on their own than when the story tells them to do so.
I've also started refining the level scale throughout the game to make it feel more welcoming and fair.

Once the escape circles are in and that I've tweaked the monsters and bosses, I'll merge back to master and release some development release so that everyone can try it out and comment. But feel free to try yourself.

Then, my next goal is to add a skill tree. And provide players with skill points they can spend on their stats and new skills instead of static skill addition and characteristic upgrades. That would be 1.1 release.

And then, of course, the next step in the story.

That's it!
Cheers,

**Comment by Bertram25** [comment]
@akien-mga FYI

**Comment by gunchleoc** [comment]
I have started playing on the new branch - this way I can test it and proofread my translation at the same time :)

I'm still at the beginning of the fighting. Maybe the first 1 or 2 encounter spots shouldn't have any spiders in them - they can cause too much damage until the party has at least 1 level and the shirt & cloak. This way, players could take a peek into the forest. Maybe set it to 2-3 green globs?

**Comment by gunchleoc** [comment]
I got an empty chest on the first forest map.

![valyria tear - empty chest](https://cloud.githubusercontent.com/assets/4095570/24090363/66eae186-0d37-11e7-8857-6ac9c5580be0.png)

**Comment by gunchleoc** [comment]
Another bug in the balance branch - do you want me to repoprt them all in this thread, or open a new issue for each one?
![valyria tear - endless percent](https://cloud.githubusercontent.com/assets/4095570/24091644/361b211a-0d41-11e7-947e-7f85cfc51757.png)

**Comment by Bertram25** [comment]
> do you want me to repoprt them all in this thread, or open a new issue for each one?

I'd say, let's discuss them first to avoid opening issues uselessly. But I won't be angry if you open issues when you feel like it. :)

> I'm still at the beginning of the fighting. Maybe the first 1 or 2 encounter spots shouldn't have any spiders in them - they can cause too much damage until the party has at least 1 level and the shirt & cloak. This way, players could take a peek into the forest. Maybe set it to 2-3 green globs?

It's completely open to discussion. I think you're right. I'd even say why not lower the spiders a bit, so that the players can get just a bit further?

> I got an empty chest on the first forest map.

That one was on purpose on Ep 1 1.0, but as we'll have to revamp the treasures content there.... If I'm not mistaken, they are almost all giving potions, which is more than sad. If you feel like parsing the list of items and discuss a plan there. I'm open. I'd rather keep elixir and advanced items such as candy, and advanced crafting material such as fur for later on, though.

> evade bug upgrade (0.00000%).

True bug, indeed.

> the victory music isn't always triggered, sometimes it's triggered late.

Do you mean the victory music is always triggered, even when you are defeated? Also, depends on what you call "late". Could you elaborate?

**Comment by gunchleoc** [comment]
> It's completely open to discussion. I think you're right. I'd even say why not lower the spiders a bit, so that the players can get just a bit further?

nerfing the spiders would make them too easy later on. I think we will be fine with just nerfing the first 2 fights by having no spiders in them. I was also thinking that it would be nice to have some pale globs in the caves rather than green globs that are a little bit stronger. They should be pretty fast to do by recoloring the graphics. Of course, some centipedes or some such that have a poison attack would add diversity.

> they are almost all giving potions, which is more than sad.

I don't see any other items that make sense at this stage... maybe a minor elixir? You can buy those at the shop anyway, so it wouldn't be overpowered. If we want something other than potions, we will need new items. Maybe some boots that give a tiny bit more armor?

Regarding the victory music, when the window comes up, there is usually some sort of fanfare music. Sometimes there is just silence instead, but if you wait doing nothing, it can be triggered eventually.

**Comment by Bertram25** [comment]
Few, I finished adding support for escape point at the logic level. And I've finally merged back to master. This way, everyone involved can test the bleeding edge, and provide feed-back, it will also be easier for the next row of changes on my todo list. ;)

**Comment by ibara** [comment]
Hi. Do not consider this me pushing you to finish version 1.1, but I am very much looking forward to it :)
I probably cannot help much, but if you need people to test on other platforms (like OpenBSD, for me) please let me know!

**Comment by gunchleoc** [comment]
I will not be able to do anything on this project before August. I'd still like to see the translation bug fixed on Windows, but I hit a brick wall and will have to try again from scratch.

**Comment by Bertram25** [comment]
Hey guys :)

Very first of all, let me take the opportunity to wish you all a great summer.
Thanks for your support @ibara . I'll indeed may need your testing help once the master branch is less... messy.
Atm, I still need to finish the big feature of the skill graph and the battle formation, and then rebalance the game to make it just playable. ;)
This is a lot already, but once it is done, I'll make a post for packaging help. Cheers,

**Comment by Bertram25** [comment]
@rydroid Hey, I've been very busy lately but will review all your job tonight. Cheers,

**Comment by Bertram25** [comment]
Some news. Takes time but I've finished a first usable version of the skill graph editor + added a first skill graph in Valyria Tear that you can navigate.

The next steps are:
- to add some nodes info for test, 
- add the logic to buy nodes with XP, 
- and replace the current logic to learn everything automatically with this.
- I'll add a link to the skill tree in the battle win screen for convenience.
Then, that will be it for the skill tree part! :D 

Afterward, I'll definitely change the story intro logic to make Bronann have a first use of the sword right away, followed by the crystal event. The hide'n'seek will become completely optional.
And then balancing, balancing, balancing.

Depending on how it goes and the gameplay needs, I may or may not add battle formation for now. We'll see.

Cheers,

**Comment by Bertram25** [comment]
Skill graph addition is now integrated back into the mainline game. (Whoohoo).
Now, it's time to revamp the intro a bit ;)

**Comment by Bertram25** [comment]
Intro is revamped. I'll fix some other linked bits now: quest log, tutorial battle position, and I'll start balancing the game afterwards :)

**Comment by Bertram25** [comment]
game intro quest log and related stuff done. Now time to balance the game XP, material and so on. :D

**Comment by Bertram25** [comment]
game intro quest log and related stuff done. Now time to balance the game XP, material and so on. :D

**Comment by Bertram25** [comment]
1.1 is basically done. I'll translate the beast into french, and then I'll package a beta :D

**Comment by gunchleoc** [comment]
So I better step up my translation effort too. Is Transifex up to date?

**Comment by Bertram25** [comment]
@gunchleoc Hey, now it is ;)

I'll do my part, fix a few things I've seen and send out a beta. But you still have time for the actual 1.1.0 release.

**Comment by Bertram25** [comment]
French translation done. I'll start working on the final small thing I've seen and packaging the beast. :D

**Comment by gunchleoc** [comment]
Maybe make an announcement on Transifex too?

**Comment by Bertram25** [comment]
> Maybe make an announcement on Transifex too?

I don't even know how to do that XD

**Comment by Bertram25** [comment]
Fixed 2 issues found by Willem (thanks!), reuploaded the fixed pot and fixed the translation both in the code and back in transifex.

**Comment by Bertram25** [comment]
Found and fixed 4 small bugs. AFAIK, I don't see big issues with the current release.
I'll wait for the translations about a week and if you don't guys don't see big issues, let's roll this one out :D

**Comment by Bertram25** [comment]
> "WELL UNDERGROUNDS" is missing from translation file
> Same for "Riverbank", "Village entrance", .. are the map names all missing?

Whoah, something must have gone wrong in the cmake target. I'll check this out and reupload the pot file once fixed. Thanks for the notice!
Opened: #643

**Comment by Bertram25** [comment]
> Found my first bug: "have go" should be "go" in this sentence. It has been fixed on Transifex, but not in current master. Am I on the correct branch?

This is weird, I'll check again

**Comment by Bertram25** [comment]
> Also, I'm not listed in the credits.

My bad ;) I'll make room for you guys ;)

**Comment by Bertram25** [comment]
- Added you and Dzmitry in the end credits
- Added again the complete translations in nl_NL and GD with the map subnames fixed.
- I also fixed the map titles issue with Ukrainian and updated the Ukrainian translation.

**Comment by Bertram25** [comment]
@gunchleoc I can't reproduce the text issue with Carson (Calum) anymore. It should be ok now.

**Comment by Bertram25** [comment]
Ok, all issues seen so far are fixed.

**Comment by gunchleoc** [comment]
> @gunchleoc I can't reproduce the text issue with Carson (Calum) anymore. It should be ok now.

Confirmed, this is fixed.

Funny bit of trivia: Carson is the Gaelic word for "Why"

**Comment by Bertram25** [comment]
> Funny bit of trivia: Carson is the Gaelic word for "Why"

Learning one word of Gaelic at a time. 🌞 

> And now we need a new .pot on Transifex ;)

Yeah XD I'll do it, but if you guys don't fix the string, I won't let it prevent a release.

**Comment by Bertram25** [comment]
Also I fixed the truncation issues as much as I could (Might still be there in specific cases for Gaelic in the skill tree, but I cannot really do much more for now, without bigger changes.)

**Comment by Bertram25** [comment]
Ok, I guess I'll wait some a final few days for maybe translation updates and then release :)

@gunchleoc I hope you enjoyed the ride. ;)

**Comment by gunchleoc** [comment]
I have, it's a pleasure working with you. Looking forward to the first RPG in my language, ever! :D

I have translated the new string, so Gaelic is ready for pulling now.

BTW please use CamelCase on my nick in the end credits: GunChleoc. It will not make sense to Gaelic speakers otherwise ;)

**Comment by Bertram25** [comment]
Cool, all fixed :)

**Comment by Bertram25** [comment]
https://github.com/ValyriaTear/ValyriaTear/releases/tag/1.1.0
https://valyriatear.blogspot.com/2019/09/episode-i-release-110-out.html
